pkg/api/plugins.go

```diff
@@ -179,6 +179,10 @@ func GetPluginMarkdown(c *m.ReqContext) Response {
 }
 
 func ImportDashboard(c *m.ReqContext, apiCmd dtos.ImportDashboardCommand) Response {
+	if apiCmd.PluginId == "" && apiCmd.Dashboard == nil {
+		return Error(422, "dashboard must be set", nil)
```

Comment:
AFAICT, our convention is to start HTTP error messages with a capital letter (like a sentence).
```suggestion
		return Error(422, "Dashboard must be set", nil)
```